ansible_rulebook/websocket.py

```diff
@@ -206,7 +232,11 @@ async def _handle_send_event_log(
 
     if logs.event:
         logger.info("Resending last event...")
-        await websocket.send(json.dumps(logs.event))
+        json_str = json.dumps(logs.event)
+        try:
+            await websocket.send(json_str)
+        except Exception as e:
```

Comment:
@bzwei Can we not use https://websockets.readthedocs.io/en/stable/reference/exceptions.html#websockets.exceptions.ConnectionClosed

Reply:
Catching and suppressing Exception is never a good idea.